problemsets-mat223/modules/module2-exercises.tex

```diff
@@ -53,7 +80,26 @@
 				parallel to $\mathcal P_2$, but which does not
 				intersect $\ell$? Why or why not?
 		\end{enumerate}
-
+        % Q4 Solutions
+        \begin{solution}
+            \begin{enumerate}
+                \item $\mathcal P_1\cap \ell = \Set*{\mat{1\\1\\0}}$
+                \item $\mathcal P_1\cap \mathcal P_2 = \Set*{s\mat{1\\1\\3} + \mat{\frac{-1}{2}\\ \frac{3}{2}\\ \frac{-1}{2}} : s \in \R}$
```

Comment:
Phrase the set `{x\in R^3: x=s*...+t*... for some s,t\in R}`. Use `-1/2` style fractions inside a vector so that it's easier to read.